Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/ObjectBindingHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Amdocs.Ginger.Common;
+using GingerCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Ginger.BusinessFlowPages_New
+{
+    public class ObjectBindingHelper
+    {
```

Comment:
i don't think this class is needed, just create Dic<Agent,SpyPage>, if you still wants to have it so call the class with more specific name like "AgentLiveSpyMap"

Reply:
As discussed renamed the class to "AgentMappingPage"